examples/README.md

```diff
@@ -0,0 +1,104 @@
+
+# Data Factory - Using the Testing Framework 
+
+To use the Testing Framework, is enough to import the published package resulting from the developement in the current repo located in src/data_factory_testing_framework.
+
+## Getting started with writting tests
+
+### Running without a Dev Container
+1. Set up an empty Python project with your favorite testing library
+2. Install the package using your preferred package manager:
+   * Pip: `pip install data-factory-testing-framework`
+   * Poetry: `poetry add data-factory-testing-framework`
+3. Start writing tests
+
+### Running with  a Dev Container
+
+#### Pre-requirements using Dev Containers
+To use a Dev Container, you need to have the following software in addition to the previous pre-requisites:
+
+- Docker
+- Visual Studio Code Remote Development Extension Pack
+
+In order to open the project in a container follow the following steps:
+
+- Open Visual Studio Code and clone the repository.
+- Hit Control-Shift-P to open the command palette and type Dev Containers: Open Folder in Container ...
+- When prompted, select the *examples* directory of the Project
+- Wait for the container to build, check the logs for more information.
+
+When the container successfully starts you can start writting your data factory pipeline tests.
+
+
+## Features - Examples
```

Comment:
I would really love to keep these features in the main README.md. As someone that finds our framework, I want them to immediately see and understand what this framework offers. These examples are also not related to any example in the examples folder btw.

I foresee the following structure eventually:
1. Main readme.md is the first page anyone sees, so I want a clear high-level explanation of why, features, some very short examples
2. framework-readme.md containing all that's relevant for developing the framework, but not for using the framework
3. getting-started.md - which is basically this / Liliams readme.md

Reply:
Yes, I don't want to take them out from the main ReadMe - I kept them there. But I would like to have also some examples in the "Getting started" readme - we might change them so we have different examples (I kept this ones for now), but I think is important to have them also here. Other than that I agree with the brakdown - we might have several iterations now because we are all trying to capture important info on the readmes

Reply:
ah got it, thanks!